### PR TITLE
Add block work score method to bindings utils

### LINKED ISSUE #2205: Expose block cost calculation to the bindings
## Description

In the explorer we would like to add the mana each block burned, and for that we would need a way to calculate the "mana block cost"

## Motivation

Above mentioned

## Requirements

Write a list of what you want this feature to do.

1. 
2. 
3. 

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

Yes/No.

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add block work score method to bindings utils

## Links to any relevant issues

Closes https://github.com/iotaledger/iota-sdk/issues/2205

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Added tests

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

File: bindings/python/iota_sdk/utils.py [hunk 1]
```diff
@@ -286,6 +286,16 @@ def shimmer_mainnet_protocol_parameters() -> ProtocolParameters:
         return ProtocolParameters.from_dict(
             _call_method('shimmerMainnetProtocolParameters'))
 
+    @staticmethod
+    def block_work_score(
+            block: Block, work_score_parameters: WorkScoreParameters) -> bytes:
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/utils.py]: bytes?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/utils.py]: No

Comment (Thoralf-M) [reply bindings/python/iota_sdk/utils.py]: Fixed, thanks

**Review by thibault-martinez — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**